src/guild/config/guildConfigs.ts

```diff
@@ -170,50 +179,21 @@ guildConfigs.set('721155702915072040', {
 	},
 });
 
-// bot testing
-guildConfigs.set('608978588976283660', {
-	id: '608978588976283660',
-	prefix: '!',
+// new bot testing
```

Comment:
What’s the use of having a bot testing server, shouldn’t you be testing things in the bot in a different isolated test setup?

Reply:
Yeh I was going to bring this up on Monday. The long goal is to have guild configs live in the database so that it can be changed on the fly without a reboot and safely splitting private config from the public codebase.

Reply:
Yeah that's something I would like to work on in the future following this PR. Currently, you can't really split local and prod configuration. I just went ahead and removed my testing guild configuration.